examples/discovery.rb

```diff
@@ -0,0 +1,4 @@
+require 'clearbit'
+require 'pp'
+
+pp Clearbit::Discovery.search(tech: 'marketo')
```

Comment:
maybe should add a note here that Discovery is a separate plan they will need to sign up for?